lib/reporters/cli/index.js

```diff
@@ -81,13 +81,80 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
         // for some reason, if there is no run summary, it is unexpected and hence don't validate this
         var run = this.summary.run;
 
+        function formatXML (inputXML, indent) {
```

Comment:
Instead of writing our own implementation, can we please explore any existing package?
This module seems good, https://github.com/chrisbottin/xml-formatter